gen/k5login_root

```diff
@@ -39,6 +43,11 @@ foreach my $resourceData (@resourcesData) {
 	}
 }
 
+foreach my $destination (@allDestinations) {
+	if (!exists $outputByDestination{$destination}) { $outputByDestination{$destination} = undef }
+}
```

Comment:
I would move this part of code into foreach through groups. Because is hard to understand from code why is it right here even if it works ok.